lib/errors/task-error.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var util = require('util');
+var defaults = require('101/defaults');
+var isObject = require('101/is-object');
+
+/**
+ * Error type for task rejection promises that indicated to the worker that
```

Comment:
non-native English speaker but not sure about this sentence. Double check that it makes sense to you